core_lib/src/interface/scribblearea.cpp

```diff
@@ -675,13 +669,8 @@ void ScribbleArea::pointerPressEvent(PointerEvent* event)
         currentTool()->pointerPressEvent(event);
     }
 
-    const bool isPressed = event->buttons() & Qt::LeftButton;
-    if (isPressed && mQuickSizing)
-    {
-        if (currentTool()->startAdjusting(event->modifiers(), 1))
-        {
-            return;
-        }
+    if (currentTool()->interruptPointerEvent(event)) {
```

Comment:
But then the function loses it's generic usage for pointer events, if we call it handleQuickSizing that just means that the BaseTool becomes tighter coupled to StrokeTool which is something i tried to avoid.

The idea with interruptPointerEvent was also that other tools could use it, so each tool can have its own control flow for when to interrupt events.